### PR TITLE
lang: [pl] add Polish translation

### PR DESCRIPTION
**Pull request by mieszkou**
I added a Polish translation of the texts

### CODE REVIEW

**Comment by datamweb** [comment]
Hi,
Thank you for contributing!
Please see https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805

File: src/Language/pl/Auth.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ */
+
+return [
+    // Exceptions
+    'unknownAuthenticator'  => '{0} nie jest poprawnym autentykatorem.',
+    'unknownUserProvider'   => 'Nie można określić User Provider do użycia.',
+    'invalidUser'           => 'Nie można zlokalizować określonego użytkownika.',
+    'bannedUser'            => 'Nie możesz cię zalogować, ponieważ jesteś obecnie zbanowany.',
+    'logOutBannedUser'      => 'Zostałeś wylogowany, ponieważ zostałeś zbanowany.',
+    'badAttempt'            => 'Nie można cię zalogować. Sprawdź swoje poświadczenia.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'badAttempt'            => 'Nie można Cię zalogować. Sprawdź swoje dane uwierzytelniające.',
```

File: src/Language/pl/Auth.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    'noPassword'            => 'Nie może zweryfikować użytkownika bez hasła.',
+    'invalidPassword'       => 'Nie można cię zalogować. Sprawdź hasło.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'invalidPassword'       => 'Nie można cię zalogować. Sprawdź swoje hasło.',
```

File: src/Language/pl/Auth.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    'noToken'               => 'Każde żądanie musi mieć token w nagłówku {0}.',
+    'badToken'              => 'Token dostępu jest nieprawidłowy.',
+    'oldToken'              => 'Token dostępu wygasł.',
+    'noUserEntity'          => 'User Entity jest niezbędne do sprawdzania poprawności hasła.',
+    'invalidEmail'          => 'Nie można zweryfikować adresu e -mail odpowiada e -maile w rekordach.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'invalidEmail'          => 'Nie można zweryfikować zgodności adresu e-mail z zarejestrowanym adresem e-mail.',
```

File: src/Language/pl/Auth.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    'unableSendEmailToUser' => 'Przepraszam, był problem z wysłaniem wiadomości e -mail. Nie mogliśmy wysłać wiadomości e -mail do "{0}".',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'unableSendEmailToUser' => 'Przepraszamy, był problem z wysłaniem wiadomości e-mail. Nie mogliśmy wysłać wiadomości e-mail do "{0}".',
```

File: src/Language/pl/Auth.php [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    'throttled'             => 'Zbyt wiele żądań złożonych z tego adresu IP. Możesz spróbować ponownie za {0} sekundy.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'throttled'             => 'Zbyt wiele żądań z tego adresu IP. Możesz spróbować ponownie za {0} sekund.',
```

File: src/Language/pl/Auth.php [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    'notEnoughPrivilege'    => 'Nie masz niezbędnego pozwolenia na wykonanie pożądanej operacji.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'notEnoughPrivilege'    => 'Nie masz uprawnień niezbędnych do wykonania żądanej operacji.',
```

File: src/Language/pl/Auth.php [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    // JWT Exceptions
+    'invalidJWT'     => 'Token jest nieważny.',
+    'expiredJWT'     => 'Token wygasł.',
+    'beforeValidJWT' => 'Token nie jest jeszcze dostępny.',
+
+    'email'           => 'Adres e -mail',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'email'           => 'Adres e-mail',
```

File: src/Language/pl/Auth.php [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    // Login
+    'login'              => 'Zaloguj sie',
+    'needAccount'        => 'Potrzebujesz konta?',
+    'rememberMe'         => 'Zapamiętaj mnie',
+    'forgotPassword'     => 'Zapomniałeś hasła?',
+    'useMagicLink'       => 'Użyj linku logowania',
+    'magicLinkSubject'   => 'Twój link logowania',
+    'magicTokenNotFound' => 'Nie można zweryfikować linku.',
+    'magicLinkExpired'   => 'Przepraszam, link wygasł.',
+    'checkYourEmail'     => 'Sprawdź swój email!',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'checkYourEmail'     => 'Sprawdź swój e-mail!',
```

File: src/Language/pl/Auth.php [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+    // Login
+    'login'              => 'Zaloguj sie',
+    'needAccount'        => 'Potrzebujesz konta?',
+    'rememberMe'         => 'Zapamiętaj mnie',
+    'forgotPassword'     => 'Zapomniałeś hasła?',
+    'useMagicLink'       => 'Użyj linku logowania',
+    'magicLinkSubject'   => 'Twój link logowania',
+    'magicTokenNotFound' => 'Nie można zweryfikować linku.',
+    'magicLinkExpired'   => 'Przepraszam, link wygasł.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'magicLinkExpired'   => 'Przepraszamy, link wygasł.',
```

File: src/Language/pl/Auth.php [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+    'checkYourEmail'     => 'Sprawdź swój email!',
+    'magicLinkDetails'   => 'Właśnie wysłaliśmy Ci wiadomość e-mail z linkiem logowania w środku. Jest on ważny tylko {0} minut.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'magicLinkDetails'   => 'Właśnie wysłaliśmy do Ciebie wiadomość e-mail z linkiem do logowania. Jest on ważny tylko {0} minut.',
```

File: src/Language/pl/Auth.php [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+    'magicLinkDisabled'  => 'Korzystanie z MagicLink jest obecnie niedozwolone.',
+    'successLogout'      => 'Udało Ci się wylogować.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'successLogout'      => 'Wylogowanie powiodło się.',
```

File: src/Language/pl/Auth.php [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+    'backToLogin'        => 'Powrót do logowania',
+
+    // Passwords
+    'errorPasswordLength'       => 'Hasła muszą mieć długość co najmniej {0, number} znaków.',
+    'suggestPasswordLength'     => 'Pass phrases - do 255 znaków - bezpieczniejsze hasła, które są łatwe do zapamiętania.',
+    'errorPasswordCommon'       => 'Hasło nie może być jednym z powszechnie używanych.',
+    'suggestPasswordCommon'     => 'Hasło sprawdzono na ponad 65 000 powszechnie używanych haseł lub haseł, które zostały wyciekane przez hacki.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'suggestPasswordCommon'     => 'Hasło zostało porównane z ponad 65 tysiącami powszechnie używanych haseł lub haseł, które wyciekły w wyniku włamania.',
```

File: src/Language/pl/Auth.php [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+    'errorPasswordPersonal'     => 'Hasła nie mogą zawierać danych osobowych.',
+    'suggestPasswordPersonal'   => 'Odmiany twojego adresu e -mail lub nazwy użytkownika nie powinny być używane do haseł.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'suggestPasswordPersonal'   => 'W hasłach nie należy stosować odmian adresu e-mail lub nazwy użytkownika.',
```

File: src/Language/pl/Auth.php [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+    'errorPasswordTooSimilar'   => 'Hasło jest zbyt podobne do nazwy użytkownika.',
+    'suggestPasswordTooSimilar' => 'Nie używaj części swojej nazwy użytkownika w hasłach.',
+    'errorPasswordPwned'        => 'Hasło {0} zostało ujawnione z powodu naruszenia danych i zostało opublikowane {1, number} razy w {2} listach haseł.',
+    'suggestPasswordPwned'      => '{0} nigdy nie powinien być używany jako hasło.Jeśli używasz go w dowolnym miejscu, zmień go natychmiast.',
+    'errorPasswordEmpty'        => 'Wymagane jest hasło.',
+    'errorPasswordTooLongBytes' => 'Hasło nie może przekraczać długości {param} bajtów.',
+    'passwordChangeSuccess'     => 'Hasło zmieniło się pomyślnie',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'passwordChangeSuccess'     => 'Hasło zostało pomyślnie zmienione',
```

File: src/Language/pl/Auth.php [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+    'userDoesNotExist'          => 'Hasło nie zostało zmienione.użytkownik nie istnieje',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'userDoesNotExist'          => 'Hasło nie zostało zmienione. Użytkownik nie istnieje',
```

File: src/Language/pl/Auth.php [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+    'resetTokenExpired'         => 'Przepraszam. Twój token resetowania wygasł.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'resetTokenExpired'         => 'Przepraszamy. Twój token resetowania wygasł.',
```

File: src/Language/pl/Auth.php [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+    // Email Globals
+    'emailInfo'      => 'Niektóre informacje o osobie:',
+    'emailIpAddress' => 'Adres IP:',
+    'emailDevice'    => 'Urządzenie:',
+    'emailDate'      => 'Data:',
+
+    // 2FA
+    'email2FATitle'       => 'Uwierzytelnianie dwuskładnikowe',
+    'confirmEmailAddress' => 'Potwierdź adres e-mail.',
+    'emailEnterCode'      => 'Potwierdź swój email',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'emailEnterCode'      => 'Potwierdź swój e-mail',
```

File: src/Language/pl/Auth.php [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+    'emailConfirmCode'    => 'Wprowadź 6-cyfrowy kod, który właśnie wysłaliśmy na Twój adres e-mail.',
+    'email2FASubject'     => 'Twój kod uwierzytelnienia',
+    'email2FAMailBody'    => 'Twój kod uwierzytelnienia jest:',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'email2FAMailBody'    => 'Twój kod uwierzytelnienia to:',
```

File: src/Language/pl/Auth.php [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+    'invalid2FAToken'     => 'Kod był nieprawidłowy.',
+    'need2FA'             => 'Musisz ukończyć weryfikację dwuskładnikową.',
+    'needVerification'    => 'Sprawdź swój e-mail, aby zakończyć aktywację konta.',
+
+    // Activate
+    'emailActivateTitle'    => 'Aktywacja e-maila',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'emailActivateTitle'    => 'Aktywacja e-mail',
```

File: src/Language/pl/Auth.php [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+    'emailActivateBody'     => 'Właśnie wysłaliśmy do Ciebie wiadomość e -mail z kodem, aby potwierdzić Twój adres e -mail.Skopiuj ten kod i wklej go poniżej.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'emailActivateBody'     => 'Właśnie wysłaliśmy do Ciebie wiadomość e-mail z kodem potwierdzającym Twój adres e-mail. Skopiuj ten kod i wklej go poniżej.',
```

File: src/Language/pl/Auth.php [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+    'invalidActivateToken'  => 'Kod był nieprawidłowy.',
+    'needActivate'          => 'Musisz zakończyć rejestrację, potwierdzając kod wysłany na adres e-mail.',
+    'activationBlocked'     => 'Musisz aktywować swoje konto przed zalogowaniem się.',
+
+    // Groups
+    'unknownGroup' => '{0} nie jest ważną grupą.',
+    'missingTitle' => 'Grupy muszą mieć tytuł.',
+
+    // Permissions
+    'unknownPermission' => '{0} nie jest ważnym pozwoleniem.',
```
Comment (michalsn) [inline src/Language/pl/Auth.php]:
```suggestion
    'unknownPermission' => '{0} nie jest ważnym uprawnieniem.',
```

**Review by michalsn — CHANGES_REQUESTED** [review]

**Comment by kenjis** [comment]
Please fix the following failure.

```
There was 1 failure:

1) Tests\Language\PolishTranslationTest::testAllIncludedLanguageKeysAreTranslated with data set "pl" ('pl')
Failed asserting that the translated language key "Auth.token" in "pl" locale differs from the original keys in the main repository.
Failed asserting that an array is empty.

/home/runner/work/shield/shield/tests/Language/AbstractTranslationTestCase.php:259
```
https://github.com/codeigniter4/shield/actions/runs/9068936216/job/24922062972?pr=1117

File: src/Language/pl/Auth.php [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
+    'invalidJWT'     => 'Token jest nieważny.',
+    'expiredJWT'     => 'Token wygasł.',
+    'beforeValidJWT' => 'Token nie jest jeszcze dostępny.',
+
+    'email'           => 'Adres e-mail',
+    'username'        => 'Nazwa użytkownika',
+    'password'        => 'Hasło',
+    'passwordConfirm' => 'Hasło (ponownie)',
+    'haveAccount'     => 'Posiadasz już konto?',
+    'token'           => 'Token',
```
Comment (kenjis) [inline src/Language/pl/Auth.php:42]: If the translation is correct, add `$excludedLocaleKeyTranslations` in the test case class.
See https://github.com/codeigniter4/shield/pull/913#issuecomment-1766414597

**Comment by kenjis** [comment]
Also, please sign all commits.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits

**Comment by datamweb** [comment]
@mieszkou thank you! 
The last thing to do is to sign all the commits.

Screenshot 2024-05-18 191831

It is explained [gpg-signing-old-commits](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits).

As a faster alternative, you can still securely sign commits without the `-S` option in git commit. See [Set Default Signing](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/signing.md#set-default-signing).

**Review by datamweb — APPROVED** [review]

**Review by kenjis — APPROVED** [review]
Thank you!

**Review by michalsn — APPROVED** [review]
Nice work!

**Comment by datamweb** [comment]
@mieszkou Welcome to Shield Contributors!
 Merged.

**Pull request merged**